autopilot/contractor.go

```diff
@@ -88,7 +106,7 @@ func (c *contractor) performContractMaintenance(cfg Config) error {
 	remaining := c.remainingFunds(cfg)
 
 	// run renewals
-	renewed, err := c.runContractRenewals(cfg, &remaining, address, toRenew)
+	renewed, isRenewed, err := c.runContractRenewals(cfg, &remaining, address, toRenew)
```

Comment:
`nit` would just build a map of `renewed` rather than return the same data twice